### DISCUSSION TITLE
How to reset code-server settings for a user so that the next startup appears to be the first one

### DISCUSSION THREAD

**Discussion by sebhahn** [coder/code-server#4940] [Q&A] [answered]
Is it possible to reset all settings and with the next startup the user has to go through the initial default screen?

**Comment by lg-nineoneninethreenine** [comment]
So are you asking about a reset process? I think it's as easy as clearing and saving the settings.json file.

**Reply by sebhahn** [reply]
Yes, I cleared that one, but the initial startup screen is not popping up

**Reply by lg-nineoneninethreenine** [reply]
Hmm, try checking on how to enable the startup screen on VS Code. Since code-server is partially based on VSCode, it might be the same result

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
The welcome screen can be activated via the menu, I think it is in the View section.

**Reply by sebhahn** [reply]
I cannot find a view for the welcome screen

**Reply by sebhahn** [reply]
OK I found it, but the initial walkthrough (Get started with VS Code ..) had to be activated manually
